packages/govuk-frontend/src/govuk/all.scss

```diff
@@ -1 +1,8 @@
+@import "settings/warnings";
```

Comment:
**nitpick** I think this will automatically be imported by `index` (via `base`, which imports `settings/index`, which imports `settings/warnings`).